lib/internal/checkUrl.js

```diff
@@ -9,9 +9,9 @@ var bhttp = require("bhttp");
 /*
 	Checks a URL to see if it's broken or not.
 */
-function checkUrl(link, baseUrl, options, callback)
+function checkUrl(link, baseUrl, options, cache, callback)
 {
-	if (typeof link==="string" || link instanceof String)
+	if (typeof link==="string" || link instanceof String === true)
```

Comment:
Please no spaces around the `===`, since it has more than one condition... one of my weird styles